.github/workflows/release.yml

```diff
@@ -414,13 +414,38 @@ jobs:
           username: ${{ secrets.DOCKER_HUB_CI_USER }}
           password: ${{ secrets.DOCKER_HUB_CI_PASSWORD }}
 
+      - name: Extract Docker metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ github.repository_owner }}/autoinstrumentation-java
+          tags: |
+            type=raw,value=${{ steps.set_version.outputs.version }}
+            type=raw,value=latest
+          labels: |
+            maintainer=swo-librarians
+            org.opencontainers.image.title=apm-java
+            org.opencontainers.image.description=Solarwinds OTEL distro Java agent
+            org.opencontainers.image.vendor=SolarWinds Worldwide, LLC
+
       - name: Build and push
         uses: docker/build-push-action@v5
         with:
           push: true
           context: agent
           platforms: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
-          tags: ${{ github.repository_owner }}/autoinstrumentation-java:${{ steps.set_version.outputs.version }},${{ github.repository_owner }}/autoinstrumentation-java:latest
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Analyze for critical and high CVEs
+        id: docker-scout-cves
+        uses: docker/scout-action@v1
+        with:
+          command: cves
+          image: "fs://./"
+          platform: "linux/amd64"
+          ignore-base: true
+          only-package-types: maven
```

Comment:
How about letting `ignore-base` be the default true and not restrict by package types. I think we should scan the entire image since that is what we provide to the customer to run in their k8s environment, and will likely give us better insight into the "heathscore" that docker hub assigns our images (an experimental feature that's currently only visible when signed in).

Reply:
I was actually thinking we can push and then scan the image, since it's not strictly just our code but an entire image that we provide to the customer. Yes it does mean having to know about vulnerabilities beyond our code but in my mind better for us to find out first rather than a customer telling us :)

Reply:
I'll add another step that scans the image which will only catch base image cves and not the jar.

Reply:
updated [run](https://github.com/solarwinds/apm-java/actions/runs/12399854801). pushing and scanning will happen during releases since we don't want to push out new code ad-hoc.